frontend/reducers/index.js

```diff
@@ -54,6 +55,11 @@ const recommendationsByType = (
         isFetching: false,
         expires: 0,
       });
+    case CLEAR_RECOMMENDATIONS:
```

Comment:
This case looks exactly like the ERROR_RECOMMENDATIONS case so you could use a fall through for this.
```
case ERROR_RECOMMENDATIONS:
case CLEAR_RECOMMENDATIONS:
      return wrapData(state, payload, {
        isFetching: false,
        expires: 0,
      });
```

Reply:
done